easybuild/easyconfigs/g/GDAL/GDAL-2.2.3-intel-2018.01-Python-3.6.3.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'ConfigureMake'
+
+name = 'GDAL'
+version = '2.2.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.gdal.org/'
+description = """GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style
+ Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model
+ to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for
+ data translation and processing."""
+
+toolchain = {'name': 'intel', 'version': '2018.01'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://download.osgeo.org/gdal/%(version)s/']
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['a328d63d476b3653f5a25b5f7971e87a15cdf8860ab0729d4b1157ba988b8d0b']
+
+dependencies = [
+    ('Python', '3.6.3'),
+    ('netCDF', '4.5.0'),
+    ('expat', '2.2.4'),
+    ('GEOS', '3.6.2', versionsuffix),
+    ('SQLite', '3.20.1'),
+    ('libxml2', '2.9.4'),
+    ('libpng', '1.6.32'),
+    ('libjpeg-turbo', '1.5.2'),
+    ('JasPer', '2.0.12'),
+    ('LibTIFF', '4.0.9'),
+    ('zlib', '1.2.11'),
+    ('cURL', '7.55.1'),
+    ('PCRE', '8.41'),
```

Comment:
@reedts existing `GDAL` easyconfigs also have a dependency on `PROJ` which is missing here?

Reply:
@boegel Maybe it makes sense to bump this PR to `intel-2018a` as it is quite old. I would then update GDAL and its dependencies to the current versions...

Reply:
@reedts Makes sense to me, the just opened #6022 could be useful here...

Reply:
@boegel Thanks! I'm on it.

Reply:
If we're not going to get this one merged, let's close it?